README.md

```diff
@@ -0,0 +1,15 @@
+# NVIDIA device plugin for Kubernetes
+
+> gh-pages branch
+
+This branch is automatically updated by GitHub actions.
+To manually update, please use the script located at `main`
+
+
+Example:
+
+```bash
+VERSION=v0.15.0
+DOWNLOAD_URL=<github release assets url>
+./hack/update-helm-index.yaml $DOWNLOAD_URL
```

Comment:
Should this refer to a shell script?

Also, I would recommend being able to also trigger this for local artifacts so that we can release this if github fails. Maybe that's a comment for #691

Reply:
Yeah, your comment makes total sense, I will modify the `update-helm-index.sh`

Reply:
`update-helm-index.sh` has been modified